dts/bindings/timer/xlnx,ttcps.yaml

```diff
@@ -0,0 +1,24 @@
+---
+title: Xilinx ZynqMP PS TTC TIMERS
+
+description: >
+    This binding gives a base representation of the Xilinx ZynqMP PS TTC TIMERS
+
+inherits:
+    !include base.yaml
+
+properties:
+    compatible:
+      constraint: "cdns,ttc"
+
+    label:
+      category: required
+
+    reg:
+      category: required
+
+    clock-frequency:
```

Comment:
Seems like this should be clock instead.

Reply:
Are you meaning to create to clock_control instance for the lpd_lsbus_clk that feeds the timers?